pkg/sql/logictest/testdata/logic_test/crdb_internal

```diff
@@ -332,11 +332,21 @@ select crdb_internal.set_vmodule('doesntexist=2,butitsok=4')
 ----
 0
 
+query T
```

Comment:
Can you add a test as an unprivileged user, where calls are denied?

Reply:
Done! Thanks for catching that.